src/StudioCore/Assets/Paramdex/ER/Defs/MapGridCreateHeightDetailLimitInfo.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PARAMDEF XmlVersion="2">
+  <ParamType>MAP_GRID_CREATE_HEIGHT_LIMIT_DETAIL_INFO_PARAM_ST</ParamType>
+  <DataVersion>2</DataVersion>
+  <BigEndian>False</BigEndian>
+  <Unicode>True</Unicode>
+  <FormatVersion>203</FormatVersion>
+  <Fields>
+    <Field Def="s32 Unknown0" />
```

Comment:
lowercase+offset, many such comments apply in places other than where they first appear